scripts/cipublish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+vagrant ssh -c "cd /vagrant
```

Comment:
Indeed. Cicero doesn't look like a good up to date example -- it doesn't implement the full suite of scripts. It looks like that cibuild script was added quickly as a quick way to get the necessary functionality there.